core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -135,7 +135,8 @@
              ng-click="showTutorCardIfScreenIsNarrow()" alt="">
 
         <div ng-if="isScreenNarrowAndShowingTutorCard()" class="supplemental-mask"></div>
-        <div class="conversation-skin-help-card" ng-if="helpCardHtml">
+          <div ng-class="{'conversation-skin-help-card help-card-standard' : !isWindowTall(),
```

Comment:
I think something is wrong with the indentation here. This div tag should be shifted 2 spaces left.